indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamSamplerSpec.java

```diff
@@ -86,140 +69,25 @@ public SeekableStreamSamplerSpec(
   public SamplerResponse sample()
   {
     final InputSource inputSource;
-    final InputFormat inputFormat;
-    if (dataSchema.getParser() != null) {
```

Comment:
this will still be needed till the time we can remove the `parser` related code as well. I suggest that we keep the adapters around for now. We could refactor the adapters such that we can still get rid of all the *firehose* code. Just enough so that people using the parser can still use the sampler.

Reply:
Thanks for the review, @abhishekagarwal87. I've addressed your concerns in the latest commit.